docs/guides/commands/samples/dependency_map_setup.cs

```diff
@@ -9,9 +9,9 @@ public class Commands
     {
         // Here, we will inject the ServiceProvider with 
         // all of the services our client will use.
-        _serviceCollection.AddSingleton(client)
-        _serviceCollection.AddSingleton(new NotificationService())
-        _serviceCollection.AddSingleton(new DatabaseService())
+        _serviceCollection.AddSingleton(client);
+        _serviceCollection.AddSingleton(new NotificationService());
+        _serviceCollection.AddSingleton(new DatabaseService());
```

Comment:
While this file is meant to be a short example, maybe add the declarations of `_serviceCollection` and `_commands`?

Reply:
Alongside this, I'd take out the class declaration and namespace imports, they don't really add anything to the example.

Reply:
I agree. I'll modify it accordingly shortly.